workers/unity/Packages/io.improbable.gdk.buildsystem/.codegen/Source/WorkerGenerationJob.cs

```diff
@@ -97,5 +86,10 @@ private List<string> ExtractWorkerTypes(string path)
             Logger.Info($"Found {workerTypes.Count} worker types:\n - {string.Join("\n - ", workerTypes)}");
```

Comment:
trace

Reply:
changed in https://github.com/spatialos/gdk-for-unity/pull/1277